sqlglot/generator.py

```diff
@@ -4041,3 +4044,18 @@ def pad_sql(self, expression: exp.Pad) -> str:
     def summarize_sql(self, expression: exp.Summarize) -> str:
         table = " TABLE" if expression.args.get("table") else ""
         return f"SUMMARIZE{table} {self.sql(expression.this)}"
+
+    def explodinggenerateseries_sql(self, expression: exp.ExplodingGenerateSeries) -> str:
+        generate_series_sql = self.sql(exp.GenerateSeries(**expression.args))
+
+        parent = expression.parent
+        if isinstance(parent, (exp.Alias, exp.TableAlias)):
+            parent = parent.parent
+
+        if self.SUPPORTS_EXPLODING_PROJECTIONS and not isinstance(parent, (exp.Table, exp.Unnest)):
+            return self.sql(exp.Unnest(expressions=[generate_series_sql]))
```

Comment:
Is it fine to pass the string / sql result in the `expressions` array here? Would it be better to build the `exp.GenerateSeries` and either pass that in `exp.Unnest` directly or generate it at the end of the function?